explorer/app/components/Table/table.tsx

```diff
@@ -46,23 +56,58 @@ export const Table = <T extends object>({
     {
       columns,
       data: items,
+      disableMultiSort: true,
       initialState: {
         pageSize: pageSize,
       } as TableState,
       manualPagination: !!pagination,
+      manualSortBy: true,
       pageCount: total,
     },
+    useSortBy,
     usePagination
   );
 
+  const handleSortClicked = (column: ColumnInstance<T>) => {
+    const newColumn =
+      column.id === sort?.sortKey && sort.sortOrder === "desc"
+        ? undefined
+        : column.id;
+    const newOrder =
+      newColumn !== sort?.sortKey
```

Comment:
Two things here:

1. Can we simplify the ternaries into if/elses? I think would help with readability here.
2. Can we detect an undefined `sort` value first up and handle that case so we don't have to add optional chaining (e.g. `sort?.`) throughout?